infrastructure/modules/glue_jobs/main.tf

```diff
@@ -43,3 +43,19 @@ resource "aws_glue_job" "transactions_cleaning" {
   default_arguments = local.transactions_cleaning_final_arguments
   timeout           = 120
 }
+
+resource "aws_glue_job" "removing_orphan_files" {
+  name     = "Orphan files removal"
+  role_arn = var.glue_role_arn
+  command {
+    name            = "glueetl"
+    script_location = "s3://${var.glue_script_bucket}/orphan_files_removal.py"
+    python_version  = "3"
+  }
+
+  worker_type       = "G.1X"
```

Comment:
I think worker_type for this purpose can be as small as possible, same with number of workers

Reply:
Worker type is small, and i can decrease number of workers but it isn't small and fast job so it may be better to stay with at least 10 of them